sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2518,4 +2518,38 @@ class HiveDDLSuite
       }
     }
   }
+
+test("SPARK-24669 Refresh table before drop database cascade") {
```

Comment:
Let's remove `SPARK-24669` prefix. We don't use JIRA ID for new feature test case. Also, please fix the indentation.